build.gradle

```diff
@@ -27,38 +41,57 @@ allprojects {
         options.encoding = 'UTF-8'
     }
 
-        //TODO: replace with GH Packages Maven repo location
     repositories {
-        ['wh-wffh-maven-local'].each { repo ->
-            maven {
-                credentials {
-                    username = findProperty('user') ?: System.getenv('user')
-                    password = findProperty('password') ?: System.getenv('password')
-                }
-
-                url "NEW_GH_PACKAGES_URL"
+        maven {
+            url = uri("https://maven.pkg.github.com/Alvearie/hri-flink-pipeline-core")
+            credentials {
+               username = findProperty("GITHUB_ACTOR") ?: System.getenv("GITHUB_ACTOR")
+               password = findProperty("GITHUB_TOKEN") ?: System.getenv("GITHUB_TOKEN")
+            }
+
+            url "https://maven.pkg.github.com/Alvearie/hri-flink-validation-fhir"
+            credentials {
+               username = findProperty("GITHUB_ACTOR") ?: System.getenv("GITHUB_ACTOR")
+               password = findProperty("GITHUB_TOKEN") ?: System.getenv("GITHUB_TOKEN")
             }
+
         }
-        mavenCentral()
         mavenLocal()
+        mavenCentral()
```

Comment:
I usually keep Maven Central last. It won't cause any issues in GitHub Action builds, but it can cause problems locally when you publish local libraries.

Reply:
Disagree.  Added a comment to note difference.